crates/ruff_linter/src/rules/flake8_pyi/snapshots/ruff_linter__rules__flake8_pyi__tests__PYI041_PYI041.pyi.snap

```diff
@@ -31,7 +30,7 @@ PYI041.pyi:24:30: PYI041 [*] Use `complex` instead of `float | complex`
 22 22 | 
 23 23 | 
 24    |-def f1(arg1: float, *, arg2: float | list[str] | type[bool] | complex) -> None: ...  # PYI041
-   24 |+def f1(arg1: float, *, arg2: list[str] | type[bool] | complex) -> None: ...  # PYI041
+   24 |+def f1(arg1: float, *, arg2: complex | list[str] | type[bool]) -> None: ...  # PYI041
```

Comment:
this is a clever way of fixing the issue, but I'm not totally sure that users will like it if the fix ends up reordering unions like this, even when the unions don't contain `None`. I think that would be pretty unexpected if I were a Ruff user.

I'm leaning towards just not providing a fix for the rule if there are multiple `None`s in the union. It feels easy to understand, and well-written code should never have multiple `None`s in a union anyway -- we even have other lints that would already detect that!